benches/bench.rs

```diff
@@ -10,7 +10,6 @@ use std::env;
 use std::fs::File;
 use std::path::{self, PathBuf};
 
-use addr2line::LookupResultExt;
```

Comment:
You need to remove this from the example too. You didn't hit it because it's #[cfg(not(unix))]